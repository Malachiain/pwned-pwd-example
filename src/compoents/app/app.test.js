import React, {Component} from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import {App} from './index'
describe('The main app',()=>{
    it('It renders the main page correctly',()=>{
        const wrapper = shallow(<App/>)
        expect(toJson(wrapper)).toMatchSnapshot();
    })
})