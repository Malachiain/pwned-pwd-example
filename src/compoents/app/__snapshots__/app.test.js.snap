// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The main app It renders the main page correctly 1`] = `
<div>
  <_default />
  <Connect(Password) />
  <_default />
</div>
`;
